akka-stream/src/main/mima-filters/2.5.10.backwards.excludes

```diff
@@ -0,0 +1,3 @@
+# #24230 stream refs - SourceRef / SinkRef
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.ActorMaterializerSettings.this")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.stage.GraphStageLogic#StageActor.this")
```

Comment:
Are both of them really safe? They seem both to touch currently public API (but maybe accidentally so).